packaging_apps_levels.md

```diff
@@ -23,22 +23,22 @@ The application installs and uninstalls correctly in all common configurations.
 The application can be updated.
 
 **Level 4**
-The application uses YunoHost users directly and allows unique identification from the YunoHost portal.
+The application can be saved and restored.
```

Comment:
Ogod I am so happy if we can change this

I am just concerned by what's the plan to handle the "migration" for old check_process files with `Level 4=0|na|1`. Do you know how / if the CI can handle that cleanly ? Or shall we edit all check_process files or something ?

Reply:
I'm going to make a pull request for package_check, it handles that part.
Still, if we indeed can ignore na here, we can't do anything about 1.

We will have to handle it manually

Reply:
We could add a version format for the checkprocess, like that we could know if level 4 is for sso/ldap or not.

Reply:
Yep, that looks like a good idea, and a good time to introduce it

Reply:
Finally done by https://github.com/YunoHost/package_check/pull/52/commits/5e726b9094301900fa3fb606cd191cf157f59206, all levels are forced to auto, except level 5.